Mod1/Labs/Week4/Debugging.md

```diff
@@ -83,7 +83,8 @@ Total: $46.64
     23> Instead of replacing `", "`, they should repleace `"\n"` to get the correct price calculation.
 
 ## Preparation
-
+<!-- Arrays moved to beginning of Collections lesson -->
```

Comment:
Love the Arrays as part of the collections lesson.  Do you think we need to keep some preparation work in here? or get rid of it?

If we do keep some prep work, maybe it could be something like asking students to answer questions about a 'stringified' list.  Like, here's a list of groceries, how would you find out what is second on the list?

Reply:
Just realized you probably are already planning something for this 🙃